src/util/util.ts

```diff
@@ -37,3 +37,15 @@ export function padNumber(value: number) {
 export function regExpEscape(text) {
   return text.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, '\\$&');
 }
+
+/**
+ * @param possibleAncestor Element to verify as being the reference element or one of its ancestors
+ * @param reference The reference element to check against
+ */
+export function isElementOrAncestorOf(possibleAncestor, reference) {
```

Comment:
Couldn't we use https://developer.mozilla.org/en-US/docs/Web/API/Node/contains instead?

Reply:
Sure, I discovered a bit later that this was used in the rest of the code. However as discussed offline indeed we don't even need to use such a feature for specific case: I can just compare the event target with the input element on which the directive is applied. For now, there is no possibility of hierarchy.